dev-client/src/components/projects/CreateProjectView/validation.ts

```diff
@@ -0,0 +1,16 @@
+import * as yup from 'yup';
```

Comment:
nit: same as prev comment, i think this is code from https://github.com/techmatters/terraso-mobile-client/pull/207 that shouldn't be here